app/code/Magento/Email/Api/BackendTemplateRepositoryInterface.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Email\Api;
+
+interface BackendTemplateRepositoryInterface
+{
+    
+    /**
+     * @param \Magento\Framework\Mail\TemplateInterface $template
+     *
+     * @return mixed
+     */
+    public function save(\Magento\Framework\Mail\TemplateInterface $template);
```

Comment:
For all new service contracts, all context arguments used in operation (like store id) should be passed explicitly